Adapter/OrmAdapter.php

```diff
@@ -70,12 +73,8 @@ public function __construct(EntityManagerInterface $em, ContentRouteEnhancer $co
     public function getLocales($contentDocument)
     {
         //todo look for better aprochement
```

Comment:
better approach is needed because this works currently just with translations managed by knpBehaviour